cmd/advertisement-operator/main.go

```diff
@@ -153,6 +154,16 @@ func main() {
 		klog.Error(err)
 		os.Exit(1)
 	}
+
+	r2 := &namectrl.NamespaceReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}
+
+	if err = r2.SetupWithManager(mgr); err != nil {
+		klog.Error(err)
+		os.Exit(1)
```

Comment:
You can use `klog.Fatal` instead of `klog.Error` and `os.Exit` jointly

Reply:
Thank you, i have changed it.